doc/languages-frameworks/julia.section.md

```diff
@@ -0,0 +1,23 @@
+# Julia {#sec-julia}
+
+### Installing Julia Packages {#sssec-installing-julia-packages}
+
+Most Julia compilers available in Nixpkgs have a passthru attribute
+called `withPackages`. This function permits creating a collection of
+packages ready for use in Julia as in many other language frameworks.
+For example, to make the package `Plots` available to Julia you can run
+
+```ShellSession
+$ nix-shell -p 'julia-bin.withPackages (p: with p; [ Plots ])'
+```
+
+The list of Julia packages in Nixpkgs is accessible through the
+compiler passthru attribute set `juliaPackages`, for example
+`julia-bin.juliaPackages`.
```

Comment:
More easily they should appear in nix search and search.nixos.org.

Reply:
How do those tools match searches?